content/src/content/jcr_root/apps/core/wcm/components/download/v1/download/download.html

```diff
@@ -21,14 +21,17 @@ <h3 class="cmp-download__title" data-sly-test.title="${download.title}" data-sly
         <a data-sly-unwrap="${!download.url}" href="${download.url}">${title}</a>
     </h3>
     <div class="cmp-download__description" data-sly-test.description="${download.description}">${description @ context="html"}</div>
-    <div data-sly-test="${(download.displayFilename && download.filename) || (download.displaySize && download.size) || (download.displayFormat && download.format)}"
-         class="cmp-download__metadata">
-        <span data-sly-test="${download.displayFilename && download.filename}" class="cmp-download__filename">${download.filename}</span>
-        <span data-sly-test="${download.displaySize && download.size}" class="cmp-download__size">${download.size}</span>
-        <span data-sly-test="${download.displayFormat && download.format}" class="cmp-download__format">${download.format}</span>
-    </div>
-    <a data-sly-unwrap="${!download.url}" class="cmp-download__action" href="${download.url}">
-        <span class="cmp-download__action-icon${download.extension ? ' cmp-download__action-icon--${download.extension}' : ''}"></span>
+    <dl class="cmp-download__metadata">
```

Comment:
@pauli I notice you removed the (long) check for has metadata. Shouldn't we hide the metadata element if no metadata is available?